src/app/[countryCode]/(main)/order/[id]/transfer/[token]/page.tsx

```diff
@@ -0,0 +1,38 @@
+import { Heading, Text } from "@medusajs/ui"
+import TransferActions from "@modules/order/components/transfer-actions"
+import TransferImage from "@modules/order/components/transfer-image"
+
+export default async function TransferPage({
+  params,
+}: {
+  params: { id: string; token: string }
+}) {
+  const { id, token } = params
```

Comment:
suggestion: I believe we should show order display_id instead of order id (at least in the header). In that case we should fetch order and display id here.